ui/espresso/IntentsAdvancedSample/app/src/androidTest/java/com/example/android/testing/espresso/intents/AdvancedSample/ImageViewHasDrawableMatcher.java

```diff
@@ -27,15 +28,15 @@
  */
 public class ImageViewHasDrawableMatcher {
 
-    public static BoundedMatcher<View, ImageView> hasDrawable() {
-        return new BoundedMatcher<View, ImageView>(ImageView.class) {
-            @Override
-            public void describeTo(Description description) {
+    public static BoundedDiagnosingMatcher<View, ImageView> hasDrawable() {
```

Comment:
Consider updating the return type to Matcher<View, ImageView>.

Reply:
@adazh 
Matcher accept only one type argument T.
Considered BoundedDiagnosingMatcher<S, T extends S> and our S is View, do I need to update the return type to Matcher<View> instead?

Thank you

Reply:
Sorry, my mistake. Yeah, should be Matcher<View>. Sorry for confusing you :).

Reply:
No problem :) Fixed!

Reply:
ok